velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.h

```diff
@@ -34,6 +40,9 @@ class HdfsFileSystem : public FileSystem {
 
  public:
   explicit HdfsFileSystem(const std::shared_ptr<const Config>& config);
```

Comment:
Can we remove the previous constructor? Is it used anywhere?

Reply:
Yes, it is useless actually. Just fixed.